manifest/3.0/3.0.8.1.xml

```diff
@@ -32,7 +32,7 @@ licenses/APL2.txt.
 
 
   <!-- Sync Gateway -->
-  <project name="sync_gateway" path="godeps/src/github.com/couchbase/sync_gateway" remote="couchbase" revision="release/3.0.8.1"/>
+  <project name="sync_gateway" path="godeps/src/github.com/couchbase/sync_gateway" remote="couchbase" revision="4c0d32181f496fdbb46c50048f8a7104001338c1"/>
```

Comment:
I think the latest commit on 3.0.8.1 is `3b4d8e6e38e357d86acd04839fe08d28c09ad97f` ?

```suggestion
  <project name="sync_gateway" path="godeps/src/github.com/couchbase/sync_gateway" remote="couchbase" revision="3b4d8e6e38e357d86acd04839fe08d28c09ad97f"/>
```